scripts/recan_bundle_tdr.py

```diff
@@ -387,7 +388,7 @@ def main(argv):
         metadata = json.load(f)
 
     dss_source = DSSSourceRef(id='',
-                              spec=SimpleSourceSpec(prefix='',
+                              spec=SimpleSourceSpec(prefix=Prefix.of_everything().effective,
```

Comment:
This is not exactly what I [asked](https://github.com/DataBiosphere/azul/pull/2931#discussion_r673377097) for. If there's a reason to deviate from that, I would like to know.

Reply:
I'm a little confused with what you're asking for. You want to add a constant attribute to Prefix that returns an instance of the prefix class, which has the prefix set to `''`. 

If the fixme does not address this we should talk during parking lot.

Reply:
I asked for a constant attribute but a method was added in response.